internal/function_window_option.go

```diff
@@ -274,6 +274,9 @@ type WindowFuncStatus struct {
 	OrderBy    []*WindowOrderBy
 }
 
+// windowNilPartitionValue Placeholder value for nil
+const windowNilPartitionValue = StringValue("^^^ZETASQLITE_NIL^^^")
```

Comment:
@goccy While exploring other issues, it seems this case comes up often.

Does it make sense to introduce a new `NilValue` class instead?

Reply:
`NULL` is not the first class's value. So I want to deal with this by handling null properly.

Reply:
```
func (s *WindowFuncStatus) Partition() (string, error) {
	partitions := make([]string, 0, len(s.Partitions))
	for _, p := range s.Partitions {
                if p == nil {
                  continue
                }
		text, err := p.ToString()
		if err != nil {
			return "", err
		}
		partitions = append(partitions, text)
	}
	return strings.Join(partitions, "_"), nil
}
```